src/app/config/selectors.js

```diff
@@ -26,18 +26,20 @@ export const getCollectionCreating = state => state.collections.isCreating;
 export const getIsUpdatingCollection = state => state.collections.isUpdating;
 export const getPreviewUsers = state => state.users.previewUsers;
 
-export const getTeams = state => state.teams.all;
-export const getTeamsLoading = state => state.teams.isLoading;
-export const getMappedTeams = createSelector(getTeams, teams => {
-    if (!teams) return [];
-    return teams.map(team => ({ id: team.id.toString(), name: team.name }));
+export const getGroups = state => state.groups.all;
+export const getGroupsLoading = state => state.groups.isLoading;
+
+export const getEnableNewSignIn = state => state.config.enableNewSignIn;
+
+export const getMappedGroups = createSelector(getGroups, getEnableNewSignIn, (groups, isNewSignIn) => {
+    if (!groups) return [];
+    return isNewSignIn
+        ? groups.map(group => ({ id: group.group_name, name: group.group_name }))
```

Comment:
I think the `name` field should have its information derived from the `description` field.
the `group_name` we are using as an ID but the `description` field is what is being used to populate the 'user-friendly' name. group_name is an autogenerated UID whereas `description` is the name the user enters when creating the team. It is worth noting, API users can create teams with no `descriptions`, because of this we should fall back and display the `group_name` as the `name` when there is no `description` to populate `name` with.

Reply:
Hi Dan, thanks. yes that is correct, I was supposed to change it after talk with James this morning, will update. Thank you